### DISCUSSION TITLE
Cartoon Network East (Not Geoblocked)

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#938] [Q&A]
### Channel ID

CartoonNetworkEast.us

### Stream URL (optional)

https://thetvapp.to/tv/cartoon-network-live-stream/

### Notes (optional)

I'm from Chile. The official streams don"t work to me.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://thetvapp.to/token/CartoonNetworkEast/ POST

necessary request to the website with a known user-agent and a cookie grabber to make the above request with a XSRF token.

**Comment by Sphinxroot** [comment]
https://www.youtube.com/watch?v=vkYIf6LpOig

https://www.youtube.com/results?search_query=cartoon+network&sp=EgJAAQ%253D%253D


https://troypoint.com/the-tv-app/

**Comment by freearhey** [comment]
The channel has been added to our blocklist due to the claims of the copyright holder: [#16839](https://github.com/iptv-org/iptv/issues/16839)
